unfoldingWord/translationCore
# Fixed bug that caused multiple copies of React being loaded

### This pull request addresses:

- Fixed bug that caused multiple copies of React being loaded


### How to test this pull request:
- Run `rm -rf node_modules` and `npm install` then `npm start`
- Open a project with **translationWords** and click the cancel button in the verse check area and then make a selection. The app should work as expected.

### Note:
```
For reusable components:
  1. Put a react dependency in both peerDependencies and devDependencies.
  2. Never put a react dependency in dependencies.

peerDependencies specifies which version(s) of React your reusable component 
supports/requires. When using npm 2 this also adds React to the list of modules 
to be installed, but this is no longer the case with npm 3.

devDependencies ensures React will be installed when you run npm install while
developing your component, or when running tests on Travis or similar.

Putting react in dependencies will cause multiple versions of React to be 
installed if somebody uses your component but has a different version of 
React in their own package.json - having multiple versions of React not only 
bloats the build, but also causes errors when different versions try to interact.
```
**Source:** https://stackoverflow.com/questions/30451556/what-is-the-correct-way-of-adding-a-dependency-to-react-in-your-package-json-for


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3687)
